lib/Models/CkanCatalogGroup.js

```diff
@@ -113,12 +113,12 @@ var CkanCatalogGroup = function(terria) {
     this.useResourceName = false;
 
     /**
-     * True to allow entire WMS servers (that is, WMS resources without a clearly-defined layer) to be
-     * added to the catalog; otherwise, false.
+     * True to allow entire WMS AND WFS servers (that is, WMS/WFS resources without a clearly-defined layer) to be
+     * added to the catalog; otherwise, false. (Yes, this is poorly named.)
      * @type {Boolean}
      * @default false
      */
-    this.allowEntireWmsServers = false;
+    this.allowEntireWmsServers = false;  // and WFS servers too.
```

Comment:
True, you are voicing something I knew to be true but which I tried to ignore... :-)
The reason I hesitated is that `CkanCatalogItem.createCatalogItemFromResource` takes an option called `allowGroups`, and I didn't want to change its interface to split out `allowWmsGroups` and `allowWfsGroups`.
So what I'll do is: change it to this, but allow the old `allowGroups` as a synonym for `allowWmsGroups`, but deprecate it.
Perfect.